slash/react/src/Loader/Loader.tsx

```diff
@@ -0,0 +1,69 @@
+import "@axa-fr/design-system-slash-css/dist/Loader/Loader.scss";
+import React from "react";
+import { getComponentClassName } from "../utilities";
+
+type LoaderMode = "none" | "get" | "post" | "delete" | "update" | "error";
+
+const getText = (index: LoaderMode) => {
+  switch (index) {
+    case "none":
+      return "";
+    case "get":
+      return "Chargement en cours";
+    case "post":
+      return "Sauvegarde en cours";
+    case "delete":
+      return "Suppression en cours";
+    case "update":
+      return "Mise à jour en cours";
+    case "error":
+      return "Une erreur est survenue lors du chargement du composant";
+    default:
+      return "";
+  }
```

Comment:
a pire si besoin d'avoir la case none on peux faire comme ca aussi
```suggestion
    case "none":
    default:
      return "";
  }
```